logstash-core/lib/logstash/agent.rb

```diff
@@ -106,7 +106,7 @@ def stop_background_services
   def configure_metric
     if collect_metric?
       @logger.debug("Agent: Configuring metric collection")
-      @metric = LogStash::Instrument::Metric.create(:root)
```

Comment:
Curious about it, As I understand we spoke about having as namespaces something like ```stats.events.in```, how does it much here the removal of ```root```? might be I lost it, but I found not references to stats anywhere :smiley_cat: